### ISSUE TITLE
Alarm vs notification

### ISSUE THREAD

**Issue by gabesbit** [drpout/boilr#137]
Hey guys,
I am new to Boilr, so feel free to correct me if I miss something.
There are 3 alert types: alarm, notification, ringtones. Problem is, notification behaves like alarm. Long vibration, long sound. This is OK for alarm but very annoying for notification. Notifcation should behave exactly like android notification. (1x vibration, 1x sound). Ideally they should take or match the system settings.

**Comment by dllud** [comment]
Thanks for the input @gabesbit

Your are right. Right now all alarms on Boilr are "annoying", i.e. they keep ringing until you turn them off. The 3 alert types have the same behaviour. That option is there just to enable choosing any sound stored on your device. It never occurred to me the interpretation you give to "alert type" but it makes sense and I bet others will think that way too.

Thus we should reduce "alert type" to 2 options:
- Alarm - Can be set to use any sound on the alarm and ringtone categories. Plays sound through Android's alarm channel (usually set to full volume even when phone is silent). Plays in loop.
- Notification - Uses sounds from the notification category. Plays sound through Android's notification channel. Plays sound only once.

**Comment by dllud** [comment]
I just gave this a second thought and found it better to keep the 3 alert types. Though, using the Notification type as @gabesbit suggested, playing sound only once.

The Ringtone type is worth to keep around because it plays sound through a different channel, the Ringtone channel, which is muted when phone is silent, unlike the Alarm channel, which is set to full volume in the normal silent mode. Thus, keeping the Ringtone type is useful for users like me which want their alarms to sound loudly during most of the day, but want them muted when phone is set to silent, such as when in a meeting or sleeping.

**Comment by algazarra** [comment]
>Notification should behave exactly like android notification. (short vibration, short sound). Ideally they should take or match the system settings.

+1

>keep the 3 alert types. Though, using the Notification type as @gabesbit suggested, playing sound only once.

+1

@gabesbit TY for the input!
@dllud that's perfect!

**Comment by refola** [comment]
I don't know if this should be a separate issue or not, but I personally find it annoying that the "notification" mode does a focus-stealing popup instead of just, you know, adding a notification to the notification drawer. I'm fine with the blinking light, but I think stealing focus is rather extreme for a notification. This is my only ongoing issue with this very convenient app.

Note: Alarm and ringtone modes steal focus too, but that's expected because alarms and calls in Android already do that; whether or not they _should_ is an Android issue where changing the status quo would probably add a new toggle-able "steal focus" permission and maybe something to convert focus-stealing in older apps into a notification.
